internal/config/types.go

```diff
@@ -136,8 +136,9 @@ type (
 	}
 
 	HostMetrics struct {
-		CollectionInterval time.Duration `yaml:"-" mapstructure:"collection_interval"`
-		InitialDelay       time.Duration `yaml:"-" mapstructure:"initial_delay"`
+		Scrapers           map[string]struct{} `yaml:"-" mapstructure:"scrapers"`
```

Comment:
Instead of a map could we do the following instead:
```
Scrapers struct {
    Cpu CpuScraper `yaml:"-" mapstructure:"cpu"`
    ...
} 

CpuScraper struct {}
```
This way each scraper has its own struct (which would be an empty struct) so that in the future we can add additional configuration if needed. Like the ones documented here: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/hostmetricsreceiver/README.md

Reply:
I see, these scrapers can also have the additional config, missed to see that before. This approach definitely makes sense.